### ISSUE TITLE
Authorization header is not allowed

### ISSUE THREAD

**Issue by ugreg** [MicrosoftDocs/powerapps-docs#396]
A number of partners and Microsoft PowerApps users have been facing this issue. 

To reproduce if I upload an API, let's say from an Open API specification like the many detailed here https://developer.fusionfabric.cloud/, there are blocking errors not allowing any of the API endpoints to be properly created because the `Authorization` header is not allowed in the custom connector definition.

Why is this? Is there any way this functionality can be adjusted, as it's expected to impact many users further using APIs like this one for FFDC when creating customer connectors.

<img width="764" alt="Screen Shot 2019-05-18 at 10 46 14 AM" src="https://user-images.githubusercontent.com/8151045/57967714-31247080-795a-11e9-9426-154f3b69eed1.png">

<img width="687" alt="Screen Shot 2019-05-18 at 12 24 59 AM" src="https://user-images.githubusercontent.com/8151045/57960941-ae70c680-7903-11e9-89d0-7b88abd4669d.png">

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c575b4c5-cfea-ab9a-842d-fb8930d3e154
* Version Independent ID: 6227aaf9-9eb8-f92b-84ae-ac205218e6b5
* Content: [Links to content about custom connectors for canvas apps - PowerApps](https://docs.microsoft.com/en-us/connectors/custom-connectors/define-openapi-definition)
* Content Source: [powerapps-docs/maker/canvas-apps/register-custom-api.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/live/powerapps-docs/maker/canvas-apps/register-custom-api.md)
* Service: **powerapps**
* GitHub Login: @lancedMicrosoft
* Microsoft Alias: **lanced**

**Comment by GavinEke** [comment]
I have the same issue using the Sophos public API. https://community.sophos.com/kb/en-us/125398

It uses both `x-api-key` and `Authorization` as the API Key in the header and the logic app custom connector won't let me validate while the Authorization header is there, even know it is a required field for the API.

**Comment by lancedMicrosoft** [comment]
Adding Sameer

**Comment by gfrsoftware** [comment]
This is also causing me issues with an API that uses a password grant_type. As the OAuth2 Generic approach does not permit password based authentication I attempted a workaround using an HTTP POST to fetch the token and then attempted to inject it into the header as an Authorization parameter only to be told that this was not allowed and I had to use OAuth2/API Key. The irony. I do not want to have to ask my users to generate their API keys via another method to just keep pasting it into the connector when the old token expires.
